NuGet.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+
+    <!-- This package source is required for package Microsoft.TargetingPack.NETFramework.v4.6.1,
+         which is not available on NuGet.org. This package contains the reference assemblies
+         required to compile for .NET Framework on Unix. -->
+    <add key="dotnet.myget.org" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+  </packageSources>
+</configuration>
```

Comment:
I'm not certain that this is the best we can do. Configuring an additional NuGet package feed at the repo level results in some packages being restored from the MyGet feed, even if they were available on nuget.org.

It would probably be better to configure this additional package source in a way such that it gets used only when restoring `BuildScripts.csproj`. I'm not sure how to do this. Moving this `NuGet.config` into `buildscripts/` does not appear to work.